### PR TITLE
Add examples for access control with organizations/teams

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -64,6 +64,7 @@ include::user:ROOT:partial$shared-nav.adoc[]
 ** xref:how-to/list-namespaces.adoc[List Namespaces]
 ** xref:how-to/using-templates.adoc[Using OpenShift Templates]
 ** xref:how-to/grant-access.adoc[Granting Access]
+** xref:how-to/fine-grained-access-examples.adoc[Fine grained access control examples]
```
Comment (corvus-ch) [inline docs/modules/ROOT/nav.adoc:67]:
```suggestion
** xref:how-to/fine-grained-access-examples.adoc[]
```

This is a shortcut for when the title of the page is the same as in the navigation.

File: docs/modules/ROOT/pages/how-to/fine-grained-access-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+├── developers (team)
+└── operations (team)
+----
+
+== Prerequisites for all examples
+
+For this guide, it's assumed that:
+
+* You are logged in to {product} using the `oc login` command.
+* You have installed `kubectl`.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/fine-grained-access-examples.adoc:24]:
```suggestion
* You have installed `kubectl`.
```

Is there a technical reason to use `kubectl`? If not, I suggest to use `oc` exclusively on this page. Or mention that `oc` and `kubectl` can (mostly) be used interchangeable.

File: docs/modules/ROOT/pages/how-to/grant-access.adoc [hunk 3]
```diff
@@ -26,3 +26,7 @@ CAUTION: Modifications to RoleBindings can prevent users from accessing their pr
 
 Access can be granted to any OpenShift Group.
 This includes that namespace owned by organization A can be managed by members of organization B or only a team of organization B.
+
+== Examples
+
+See xref:how-to/fine-grained-access-examples.adoc[Fine grained access control examples] for more detailed examples.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/grant-access.adoc:32]:
```suggestion
See xref:how-to/fine-grained-access-examples.adoc[] for more detailed examples.
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]
Please clarify on the need for kubectl.

File: docs/modules/ROOT/pages/how-to/fine-grained-access-examples.adoc (hunk 2 above)
Comment (bastjan) [reply docs/modules/ROOT/pages/how-to/fine-grained-access-examples.adoc]: there's none. i switch the tutorial to `oc` only.

**Pull request merged**